person.rb

```diff
@@ -28,11 +31,8 @@ def of_age?
   def can_use_services?
     of_age? || @parent_permission
   end
-end
 
-person = Person.new(22, 'maximilianus')
-person.correct_name
-capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] In this method, the functionality to create a rental is already domiciled in the constructor of the `Rental` class. Simply ensure that your method accepts a `book` parameter, then you initialize a new `Rental` class as shown below 👇 This is important because a rental belongs to a person and also belongs to a book
#### DO THIS INSTEAD
```ruby
def add_rental(book, date)
  Rental.new(book, self, date)
end
```